runner/src/elf2tab.rs

```diff
@@ -4,9 +4,32 @@ use std::io::ErrorKind;
 use std::path::PathBuf;
 use std::process::Command;
 
+fn get_platform_architecture(platform: &str) -> Option<String> {
```

Comment:
You don't need to allocate here -- just return `Option<&'static str>`.

Reply:
That is what initially did, but as I wanted to support the overriding of the architecture via a command line flag, it had to be string (the flag is String and it would allow me to use .or_then). As I am removing the flag, I will change it back to &'static str.